test/onnx/verify/matmulinteger_uns_test.cpp

```diff
@@ -43,6 +43,6 @@ TEST_CASE(matmulinteger_uns_test)
     auto result = p.eval(pp).back();
     std::vector<int32_t> result_vector;
     result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
-    std::vector<int32_t> gold = {34, 97, 28, 82, 22, 67, 16, 52};
+    std::vector<int32_t> gold = {45730, 44641, 46108, 45010, 46486, 45379, 46864, 45748};
```

Comment:
Can you make sure these golden data are correct by running them through numpy python code ?

Reply:
You can use 
https://github.com/onnx/onnx/blob/main/onnx/reference/ops/op_matmul_integer.py#L15